.travis.yml

```diff
@@ -1,29 +1,24 @@
+dist: focal
```

Comment:
Travis runs on an old version (16) of Ubuntu by default, and some things didn't work with these changes on this, this switches to Ubuntu 20 which seems to work fine

Reply:
I don't actually know if you're still using Travis - the latest build I see linked from the README is https://travis-ci.org/github/dstufft/django-passwords/builds/751467158, which failed a year ago, and I don't see CI setup for PRs - possibly this was dropped when Travis scaled back their free plans? In any case, these changes should improve the build if you do want to use this, or could at least serve as a starting point to switch to using GitHub actions or similar in future